webdriver/tests/sessions/new_session/support/create.py

```diff
@@ -6,6 +6,8 @@
     ("platformName", [None]),
     ("pageLoadStrategy", ["none", "eager", "normal", None]),
     ("proxy", [None]),
+    ("timeouts", [{"script": 0, "pageLoad": 2.0, "implicit": 2**64 - 1},
+                  {"script": 50, "pageLoad": 25}, {"script": 500}]),
```

Comment:
This is a bit of nit-picking, but can you put the last `{"script": 500}` entry on a separate line so it is clear there are three tests here?

Reply:
Sure!